src/webgl-surface/drawing/shape/curved-line-shape.ts

```diff
@@ -228,4 +251,45 @@ export class CurvedLineShape<T> extends CurvedLine<T> {
     super.setPoints(start, end, controlPoints);
     this.cachedQuadSegments = [];
   }
+
+  containsPoint(point: IPoint) {
+      if (this.type === CurveType.CircularCW || this.type === CurveType.CircularCCW) {
+
+        // Center
+        const center = this.controlPoints[0];
```

Comment:
The true center of a curved line for CW or CCW arc lines is stored in controlPoints[1]. For single control Bezier curves this is probably still controlPoints[0].